<launch>
    <node pkg="prometheus_detection" type="aruco_det" name="aruco_det" output="screen">
        <param name="camera_topic" type="string" value="/prometheus/camera/rgb/image_raw" />
        <param name="camera_parameters" type="string" value="$(find prometheus_detection)/shell/calib_webcam_640x480.yaml" />
        <param name="output_pose_topic" type="string" value="/prometheus/object_detection/aruco_det" />
        <param name="output_multi_pose_topic" type="string" value="/prometheus/object_detection/multi_aruco_det" />
        <param name="output_image_topic" type="string" value="/prometheus/camera/rgb/image_aruco_det" />
        <param name="dictionary_type" type="int" value="6" />
        <param name="binding_id" type="int" value="1" />
        <param name="target_marker_length" type="double" value="0.025" />
    </node>
</launch>
